.github/ISSUE_TEMPLATE.md

```diff
@@ -13,20 +13,25 @@ is the best place for getting support.
 
 [Description of the issue]
 
-### Steps to Reproduce (if issue is bug)
+#### Expected behavior (if applicable):
```

Comment:
✏️ Maybe we could add a section here with a set of checkboxes here for very broad types of issue:
* Bug report
* Dependency upgrade
* Documentation change
* Enhancement
* New Feature

Reply:
These checkboxes could roughly correspond to sections below that we expect to be filled out in some cases, maybe.

Reply:
TIL you could have multiple issue templates on GH. 💯

Reply:
Same! Would we want to have a different issue template per type? I think that would still be overlap with features, enhancements as well as overlap between dependency upgrades and documentation.

Reply:
As a side note should with this knowledge should we create a PR template for back ports/ forward ports or leave it as it is?